src/renderers/dom/fiber/__tests__/ReactDOMFiber-test.js

```diff
@@ -81,6 +81,21 @@ describe('ReactDOMFiber', () => {
   });
 
   if (ReactDOMFeatureFlags.useFiber) {
+
+    it('can import findDOMNode in Node environment.', () => {
+      const previousRAF = global.requestAnimationFrame;
+      const previousRIC = global.requestIdleCallback;
+      try {
+        global.requestAnimationFrame = undefined;
+        global.requestIdleCallback = undefined;
+        jest.resetModules();
+        require('ReactDOM');
```

Comment:
Maybe they’re failing because they don’t expect a new `resetModules` in an arbitrary test? Perhaps it’s better to keep these two tests in a different file then, but keep them wrapped in the `useFiber` check.